lib/services.credentials.js

```diff
@@ -184,14 +260,13 @@ bedrock.events.on('bedrock-express.configure.routes', function(app) {
     // public         F/-             F/-               -/F
     // partial-public F/-             P/-               -/P
     // private        F/-             E/-               -/E
-    // FIXME
+    // FIXME: no actor used when getting credential
     var actor = (req.user && req.user.identity) ? req.user.identity : null;
```

Comment:
Note that a `null` actors means "do not check permissions" ... and an `undefined` actor means "deny access to everything, the actor is anonymous`.